plugins/outputs/plugin.go

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2022 Aspect Build Systems, Inc.
+ *
+ * Licensed under the aspect.build Commercial License (the "License");
+ * you may not use this file except in compliance with the License.
+ * Full License text is in the LICENSE file included in the root of this repository.
+ */
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	goplugin "github.com/hashicorp/go-plugin"
+
+	"aspect.build/cli/pkg/bazel"
+	"aspect.build/cli/pkg/interceptors"
+	"aspect.build/cli/pkg/ioutils"
+	"aspect.build/cli/pkg/plugin/sdk/v1alpha2/config"
+	aspectplugin "aspect.build/cli/pkg/plugin/sdk/v1alpha2/plugin"
+)
+
+func main() {
+	goplugin.Serve(config.NewConfigFor(NewDefaultPlugin()))
+}
+
+type Outputs struct {
+	aspectplugin.Base
+	ioutils.Streams
+	Bzl bazel.Bazel
+}
+
+func NewDefaultPlugin() *Outputs {
+	return New(ioutils.DefaultStreams, bazel.New())
+}
+
+func New(streams ioutils.Streams, bzl bazel.Bazel) *Outputs {
+	return &Outputs{
+		Streams: streams,
+		Bzl:     bzl,
+	}
+}
+
+func (plugin *Outputs) CustomCommands() ([]*aspectplugin.Command, error) {
+	return []*aspectplugin.Command{
+		aspectplugin.NewCommand(
+			"outputs",
+			"Inspect declared outputs.",
+			"Queries for the outputs declared by actions generated by the given target.",
+			func(ctx context.Context, args []string, bzl bazel.Bazel) error {
+				workspaceRoot := ctx.Value(interceptors.WorkspaceRootKey).(string)
+				bzl.SetWorkspaceRoot(workspaceRoot)
```

Comment:
This is done for you behind the scenes. `bzl` should come into the function ready to go. Happens here: 
https://github.com/aspect-build/aspect-cli/blob/main/pkg/plugin/sdk/v1alpha2/plugin/interface.go#L135

Reply:
oh that's great, since I'm sure plugin authors wouldn't remember to do that and would get wrong bazel versions